.travis.yml

```diff
@@ -2,5 +2,4 @@ language: clojure
 lein: lein2
 script: lein2 test-all
 jdk:
-  - openjdk7
-  - openjdk6
+  - oraclejdk8
```

Comment:
I can't figure out how to get rhino to be pulled in & be usable. Doo seems to depend on rhino being installed globally, nashorn works because it is sorta-global.

I've opened an issue on the doo tracker: https://github.com/bensu/doo/issues/84 seeking pointers on how to get doo to run with the local dependency and not require the global install.